src/components/Post.js

```diff
@@ -62,7 +62,7 @@ export class Post extends React.Component {
 
   static contextTypes = {currentUser: object}
 
-  static childContextTypes = {post: object}
+  static childContextTypes = {post: object, dispatch: func, currentUser: object}
```

Comment:
Surely not necessary at all, not sure why I ended-up adding that here. Correcting.